app/controllers/api/v2/hosts_controller.rb

```diff
@@ -227,6 +227,15 @@ def disassociate
         render 'api/v2/hosts/show'
       end
 
+      api :POST, "/hosts/:id/forget_status", N_("Forget a particular host sub-status")
```

Comment:
Would a more restful way be `DELETE /hosts/:id/statuses/:status`

Reply:
Actually I didn't realize it before but `/hosts/:id/status/:type` exists.  Perhaps I could do a `DELETE` to that?